packages/neutron-sdk/src/bindings/msg.rs

```diff
@@ -239,28 +252,61 @@ impl CustomMsg for NeutronMsg {}
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
-/// Describes response structure for **RegisterInterchainQuery** msg
+/// Describes response structure for **RegisterInterchainQuery** msg.
 pub struct MsgRegisterInterchainQueryResponse {
-    /// **id** is an identifier of newly registered interchain query
+    /// **id** is an identifier of newly registered interchain query.
     pub id: u64,
 }
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
-/// MsgSubmitTxResponse defines the response for Msg/SubmitTx
+/// MsgSubmitTxResponse defines the response for Msg/SubmitTx.
 pub struct MsgSubmitTxResponse {
     /// **sequence_id** is a channel's sequence_id for outgoing ibc packet. Unique per a channel.
     pub sequence_id: u64,
-    /// **channel** is a src channel on neutron side trasaction was submitted from
+    /// **channel** is a src channel on neutron side trasaction was submitted from.
     pub channel: String,
 }
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
-/// MsgSubmitTxResponse defines the response for Msg/IbcTransfer
+/// MsgSubmitTxResponse defines the response for Msg/IbcTransfer.
 pub struct MsgIbcTransferResponse {
     /// **sequence_id** is a channel's sequence_id for outgoing ibc packet. Unique per a channel.
     pub sequence_id: u64,
-    /// **channel** is a src channel on neutron side trasaction was submitted from
+    /// **channel** is a src channel on neutron side trasaction was submitted from.
     pub channel: String,
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+/// AdminProposal defines the struct for various proposals which Neutron may accept.
```

Comment:
/// AdminProposal defines the struct for various proposals which Neutron may accept.

--->

/// AdminProposal defines the struct for various proposals which Neutron **'s adminmodule** may accept.

? just to be more specific